### PR TITLE
assignment 2 - z170095

### PR DESCRIPTION
**Pull request by hxshfx**

### CODE REVIEW

File: Assignment2/assignment 2 - z170095.ttl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+:Class01 a :Class ;
+    :includes :Sensor029 ;
+    :includes :Computer101 .
+
+:Sensor029 a :Sensor ;
+    :hasMeasurement :Measurement8401 .
+
+:Measurement8401 a :Measurement ;
+    :hasTemperature 29^^xsd:integer ;
+    :atTime "2010-06-12T12:00:12"^^xsd:date .
```
Comment (ocorcho) [inline Assignment2/assignment 2 - z170095.ttl:29]: It should be a dateTime, not just date.

**Review by ocorcho — CHANGES_REQUESTED** [review]
Be careful, since you are modifying the assignment 1. Fix it before submitting, by syncing your repo with the latest version of this one.
As for the RDF file, I like the fact that you have also defined classes

File: Assignment2/assignment 2 - z170095.ttl (hunk 1 above)
Comment (hxshfx) [reply Assignment2/assignment 2 - z170095.ttl]: Thank's for your feedback! I messed up a little bit with the git commands, so I'm sending a new pull request from a new local repository. Maybe should be this request closed?
And about the RDF file, I also think I'm not using prefixes correctly, but I'm sure we will know more about these details in next lectures.

**Comment by ocorcho** [comment]
Ok, so I will be closing this one then.

**Pull request closed without merging**